### ISSUE TITLE
Discovery: What is the impact of CSS on va.gov on page load 

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/va.gov-team#16887]
## Issue Description
Discovery: What is the impact of CSS on va.gov on page load. Currently, our understanding is that loading CSS is slowing down page load.  

---
## Tasks
- [ ] Investigate CSS page load

## Acceptance Criteria
- [ ] Identify the impact of loading CSS on the page load. 

---

**Comment by cvalarida** [comment]
# TL;DR
Our action items:
- Reduce the size of `style.css` by [removing unused styles with PurgeCSS](https://github.com/department-of-veterans-affairs/va.gov-team/issues/17593)
- Make a determination of whether we can render before we finish loading CSS
    - This will cause a flash of unstyled text

# The full story
## A broad look
According to [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/?url=va.gov), the First Contentful Paint (FCP) occurs on average at about 2.1 seconds after the page is requested. There are a lot of other problems that need to be tackled, but FCP is one that's directly affected by `style.css`, which is where we load up our design system.

![image](https://user-images.githubusercontent.com/12970166/102560033-1b22e880-4086-11eb-9284-85f49a7d3514.png)

## First Contentful Paint
The FCP only happens after `style.css` is loaded because it's [render-blocking](https://developers.google.com/web/fundamentals/performance/critical-rendering-path/render-blocking-css).
![image](https://user-images.githubusercontent.com/12970166/102560646-9507a180-4087-11eb-872e-e77c0d0ed265.png)

### What can we do about this?
We have two (non-exclusive) options:
1. Reduce the size of `style.css`
    - There's an excellent tool called `PurgeCSS` that does this
2. Make loading `style.css` not block rendering
    - There are some media type tricks we can do, but it'll cause a flash of unstyled text
    - We'd need to determine whether that's okay or not

**Comment by cvalarida** [comment]
I'm considering this discovery done for now. We have a path forward for how to improve site load performance [in this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/17593#issuecomment-748370271).
